tests/testthat/test-one_minus_kaplan_meier.R

```diff
@@ -0,0 +1,13 @@
+test_that("one_minus_kaplan_meier works as expected", {
```

Comment:
can you add a second test that looks into the edge case where you 0 rows and just return 0?